samples/js-tests/src/CocoStudioTest/GUITest/UIBaseLayer.js

```diff
@@ -49,6 +49,18 @@ var UIBaseLayer = cc.Layer.extend({
         this._mainNode = mainNode;
         this._topDisplayText = topDisplayText;
     },
+
+    _parseUiFile: function(file){
```

Comment:
Please rename it to `_parseUIFile`, `UI` stands for User Interface, so both U and I should be capital